innsending/src/main/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/innsending/InnsendingService.kt

```diff
@@ -120,6 +120,11 @@ class InnsendingService(
         val forespoerselId = Key.FORESPOERSEL_ID.les(UuidSerializer, melding)
         val skjema = Key.SKJEMA_INNTEKTSMELDING.les(Innsending.serializer(), melding)
 
+        if (Key.FORESPOERSEL_SVAR in melding.keys) {
+            // Logger for å danne et bilde av tiden det tar å hente forespørsler.
+            logger.info("InnsendingService: forespørsel svar mottatt")
+        }
```

Comment:
Denne vil potensient logge flere ganger fordi `melding` vil inneholde nøkkelen hver gang etter forespørselen er hentet.

Reply:
Ja, jeg så det når jeg testa i dev nå, men tenker at det ikke gjør så mye hvis jeg bare ser på den første av disse loggene når jeg skal gjøre stikkprøvene mine. Er du enig? 😄 

Og så tenker jeg å fjerne hele loggingen når jeg lager PRene med tiltakene for å få innsending raskere.

Reply:
Funker for meg!